src/pyfaf/actions/assign_release_to_builds.py

```diff
@@ -83,6 +83,7 @@ def run(self, cmdline, db):
                 if found_build:
                     self._add_into_build_opsysrelease_arch(db, found_build,
                                                            opsysrelease, arch)
+                db.session.flush()
```

Comment:
Shouldn't this be one level up? It is in the for loop, wasn't the point of this commit to have flush outside loops?